src/main/scala/org/hammerlab/magic/test/listener/TestSparkListener.scala

```diff
@@ -158,7 +158,12 @@ object TestSparkListener {
 
   var instance: TestSparkListener = _
 
-  def apply(): TestSparkListener = instance
+  def apply(): TestSparkListener = {
+    if (instance == null) {
+      instance = new TestSparkListener()
```

Comment:
Well, I thought about adding it instead using `sc.addSparkListener`, but there was not any way of checking if listener has been set already. Anyway, will fix.